src/format/KeePass2.cpp

```diff
@@ -133,3 +129,29 @@ KeePass2::ProtectedStreamAlgo KeePass2::idToProtectedStreamAlgo(quint32 id)
         return KeePass2::ProtectedStreamAlgo::InvalidProtectedStreamAlgo;
     }
 }
+
+QString KeePass2::cipherToString(QUuid cipherUuid)
+{
+    if (cipherUuid == KeePass2::CIPHER_AES256) {
+        return QObject::tr("AES 256-bit");
+    } else if (cipherUuid == KeePass2::CIPHER_TWOFISH) {
+        return QObject::tr("Twofish 256-bit");
+    } else if (cipherUuid == KeePass2::CIPHER_CHACHA20) {
+        return QObject::tr("ChaCha20 256-bit");
+    }
+    return QObject::tr("Invalid Cipher");
+}
+
+QString KeePass2::kdfToString(QUuid kdfUuid)
```

Comment:
Optional: we can rename the input argument from `kdfUuid` to just `uuid` because it is obvious for `kdfToString` method.

Reply:
I mean in scope of this method it is obvious the expected given uuid is kdf one and no other uuids in the method we have. But if you have a global `uuid` it may conflict, sure thing.
Any way, it was just a cosmetic suggestion.